css/global.css

```diff
@@ -0,0 +1,23 @@
+.jobs-unified-top-card__subtitle-secondary-grouping
```

Comment:
Created this single file because it was unclear if the view.css and search.css files were conflicting.